jmeter/src/main/java/org/eclipse/hono/jmeter/ui/HonoSenderSamplerUI.java

```diff
@@ -65,7 +67,9 @@ public HonoSenderSamplerUI() {
         assertion = new JLabeledTextField("Registration Assertion");
         contentType = new JLabeledTextField("Content type");
         data = new JLabeledTextArea("Message data");
-        waitForCredits = new JCheckBox("Wait for credits");
+        waitForCredits = new JCheckBox("Wait for credits if none is left");
+        waitForDeliveryResult = new JCheckBox("Wait for delivery result");
+        waitForDeliveryResult.setToolTipText("<html>Deselecting this option increases throughput, especially of <em>event</em> messages.</html>");
```

Comment:
however, in particular for events you will usually **not** want to disable this. However, the way it is written, it sounds like *good tuning advice* which is quite misleading FMPOV ...